templates/bridgeserver2.yaml

```diff
@@ -380,6 +380,9 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: synapse.oauth.client.secret
           Value: !Ref SynapseOAuthClientSecret
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: crc.geocode.api.key
+          Value: !Ref CrcGeoCodeApiKey
```

Comment:
you need to add `CrcGeoCodeApiKey` to the `Parameters:` section of this file

Reply:
Added without echo to terminal.